Lantern doesn't run on XP (sometimes)

Seems caused by **panicwrap** package. Getting rid of it seems to fix it. There are two options here:

* Remove panicwrap usage under Windows XP (not very satisfactory given the amount of users that run this OS).
* Try to fix panicwrap itself (may be due to an OS limitation or a Go bug, or very time-consuming to catch).

The first issue, referenced in #3086 was:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x53633b]

goroutine 484 [running]:
sync/atomic.LoadUint64(0x117acec4, 0x53aa1a, 0xc)
        /go/src/sync/atomic/asm_386.s:165 +0xb
github.com/getlantern/detour.(*Conn).anyDataReceived(0x117aceb0, 0x117c8f40)
        /lantern/src/github.com/getlantern/detour/detour.go:199 +0x32
github.com/getlantern/detour.func┬╖008()
        /lantern/src/github.com/getlantern/detour/detour.go:145 +0xf2
created by github.com/getlantern/detour.func┬╖010
        /lantern/src/github.com/getlantern/detour/detour.go:150 +0x63f

goroutine 1 [syscall, locked to thread]:
syscall.(*Proc).Call(0x1162d980, 0x115dff74, 0x2, 0x2, 0x7565a0, 0x8d3134, 0x0, 0x0)
        /go/src/syscall/dll_windows.go:132 +0x27b
syscall.(*LazyProc).Call(0x11557a60, 0x115dff74, 0x2, 0x2, 0x0, 0x0, 0x0, 0x0)
        /go/src/syscall/dll_windows.go:279 +0x5f
github.com/getlantern/systray.nativeLoop()
        /lantern/src/github.com/getlantern/systray/systray_windows.go:49 +0xba
github.com/getlantern/systray.Run(0x8d31ec)
        /lantern/src/github.com/getlantern/systray/systray.go:56 +0x82
main.runOnSystrayReady(0x8d31ec)
        /lantern/src/github.com/getlantern/flashlight/systray.go:14 +0x28
main.main()
        /lantern/src/github.com/getlantern/flashlight/flashlight.go:129 +0x4c
```

The second is:
```
Sep 02 18:41:29.169 - 0m35s ERROR flashlight.client: server.go:1777 http: panic serving 127.0.0.1:4239: runtime error: invalid memory address or nil pointer dereference
goroutine 3052 [running]:
net/http.func·011()
        /go/src/net/http/server.go:1130 +0xa8
io.Copy(0x31595f10, 0x12173368, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/src/io/io.go:362 +0x17f
net.genericReadFrom(0x31581198, 0x11f5aa60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/src/net/net.go:404 +0x92
net.(*TCPConn).ReadFrom(0x11f5aa60, 0x0, 0x0, 0x315856c8, 0x11f5aa60, 0x0, 0x0)
        /go/src/net/tcpsock_posix.go:72 +0xbc
io.Copy(0x31581198, 0x11f5aa60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/src/io/io.go:358 +0xf4
github.com/getlantern/flashlight/client.pipeData(0x31580ac0, 0x11f5aa60, 0x0, 0x0, 0x11d7dce0)
        /lantern/src/github.com/getlantern/flashlight/client/handler.go:136 +0x155
github.com/getlantern/flashlight/client.(*Client).intercept(0x1155c780, 0x315855a8, 0x1158b730, 0x11a34a80)
        /lantern/src/github.com/getlantern/flashlight/client/handler.go:105 +0x596
github.com/getlantern/flashlight/client.(*Client).ServeHTTP(0x1155c780, 0x315855a8, 0x1158b730, 0x11a34a80)
        /lantern/src/github.com/getlantern/flashlight/client/handler.go:30 +0x14e
net/http.serverHandler.ServeHTTP(0x116a9980, 0x315855a8, 0x1158b730, 0x11a34a80)
        /go/src/net/http/server.go:1703 +0x145
net/http.(*conn).serve(0x11859440)
        /go/src/net/http/server.go:1204 +0x9d8
created by net/http.(*Server).Serve
        /go/src/net/http/server.go:1751 +0x2ce
```

--- a/src/github.com/getlantern/flashlight/flashlight.go
+++ b/src/github.com/getlantern/flashlight/flashlight.go
@@ -113,11 +113,15 @@ func main() {
 	if err != nil {
 		// Something went wrong setting up the panic wrapper. This won't be
 		// captured by panicwrap
+		// At this point, continue execution without panicwrap support. There
+		// are known cases where panicwrap will fail to fork, such as Windows
+		// GUI app
 		log.Errorf("Error setting up panic wrapper: %v", err)
-	}
-	// If exitStatus >= 0, then we're the parent process.
-	if exitStatus >= 0 {
-		os.Exit(exitStatus)
+	} else {
+		// If exitStatus >= 0, then we're the parent process.
+		if exitStatus >= 0 {
+			os.Exit(exitStatus)
+		}
 	}
 
 	parseFlags()

